src/lib/optimizer/chunk_statistics/chunk_statistics.hpp

```diff
@@ -15,16 +15,19 @@ namespace opossum {
  */
 class ChunkStatistics final : public std::enable_shared_from_this<ChunkStatistics> {
  public:
-  explicit ChunkStatistics(std::vector<std::shared_ptr<ChunkColumnStatistics>> statistics) : _statistics(statistics) {}
+  explicit ChunkStatistics(std::vector<std::shared_ptr<ChunkColumnStatistics>> chunk_column_statistics)
+      : _chunk_column_statistics(chunk_column_statistics) {}
 
-  const std::vector<std::shared_ptr<ChunkColumnStatistics>>& statistics() const { return _statistics; }
+  const std::vector<std::shared_ptr<ChunkColumnStatistics>>& chunk_column_statistics() const {
+    return _chunk_column_statistics;
+  }
 
   /**
    * calls can_prune on the ChunkColumnStatistics corresponding to column_id
    */
   bool can_prune(const ColumnID column_id, const AllTypeVariant& value, const PredicateCondition predicate_type) const;
 
  protected:
-  std::vector<std::shared_ptr<ChunkColumnStatistics>> _statistics;
+  std::vector<std::shared_ptr<ChunkColumnStatistics>> _chunk_column_statistics;
```

Comment:
That does not really make it better, doesn't it?

Reply:
Better than `_statistics` for sure. `_statistics` as a name sounds like an illness.

Reply:
Any better ideas?

I know statistics vs. data statistics vs. aggregates can pretty much mean the same. I would not mind renaming those "statistics" to aggregates and clarify all stored "data statistics" by renaming them to some kind of "aggregates".

What's your take @mrzzzrm ? @mrks & @Bensk1 ?

Reply:
Didn't want to spark a naming discussion, and since this fixes a blocker for #715 I reverted the naming